.github/ISSUE_TEMPLATE/stabilization-review.md

```diff
@@ -0,0 +1,50 @@
+---
+name: TC Review Request
+about: Request a Technical-Committee stability review (promotion to Stable/GA)
```

Comment:
Should we avoid the term "GA" here? IIRC this is the place where GA is defined https://github.com/open-telemetry/opentelemetry-specification/blob/74c626723da290127a1d9d6bb18f9117235e7f10/specification/versioning-and-stability.md?plain=1#L429 and it doesn't seem to be something the TC review is trying to cover.